src/python/foglamp/core/routes.py

```diff
@@ -32,23 +32,22 @@ def setup(app):
 
     # Scheduler
     # Scheduled_processes - As per doc
-    app.router.add_route('GET', '/foglamp/scheduled/processes', api_scheduler.get_scheduled_processes)
-    app.router.add_route('GET', '/foglamp/scheduled/process/{scheduled_process_name}', api_scheduler.get_scheduled_process)
+    app.router.add_route('GET', '/foglamp/schedule/process', api_scheduler.get_scheduled_processes)
+    app.router.add_route('GET', '/foglamp/schedule/process/{scheduled_process_name}', api_scheduler.get_scheduled_process)
```

Comment:
Required as we need to verify on server side a valid process name in creating a schedule.